OtripleS.Web.Api/Services/Foundations/Guardians/GuardianService.Validations.cs

```diff
@@ -198,13 +198,6 @@ private static void ValidateAgainstStorageGuardianOnModify(Guardian inputGuardia
                         parameterValue: inputGuardian.UpdatedDate);
             }
         }
-
-        private void ValidateStorageGuardians(IQueryable<Guardian> storageGuardians)
-        {
-            if (!storageGuardians.Any())
-            {
-                this.loggingBroker.LogWarning("No guardians found in storage.");
-            }
-        }
+        
```

Comment:
Remove this extra empty line

Reply:
The extra empty line has been removed as identified & suggested!